recipes/paicos/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "paicos" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/paicos-{{ version }}.tar.gz
+  sha256: ce29e503ec6d3e876a0481e94afd46470195288fb0f0628cc10031be48418e78
+
+build:
+  skip: True  # [py<38]
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - cross-python_{{ target_platform }}    # [build_platform != target_platform]
+    - python                                # [build_platform != target_platform]
+    - cython                                # [build_platform != target_platform]
+    - numpy                                 # [build_platform != target_platform]
+  host:
+    - python
+    - cython >=0.29.21
+    - numpy >=1.20
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - conda-forge/linux-64::_openmp_mutex ==4.5=2_gnu  # [linux]
+    - scipy
+    - {{ pin_compatible("numpy") }}
+    - h5py
+    - astropy
+    - numba
+    - matplotlib-base
+    - cython
+    - pytest
+    - pytest-order
```

Comment:
Thanks for the feedback on the recipe @xhochy!

I tried removing these from the recipe, as suggested, but the tests then fail. These packages are required for building the package (cython) and for running tests (pytest and pytest-order). I'd like to keep them in my requirements.txt because I think removing from there will mess up my pypi package and/or change instructions for compiling from source.

I'd like to keep cython in, as I would like to ship the package with the .pyx files, and I'd like to keep the pytest dependency, as I would like the user to be able to run tests.

Reply:
Normally, these are not listed as explicit runtime dependencies and are only installed if required to keep the runtime environment as small as possible. You also have them declared as upstream dependencies and thus I would be open to merge this as-is but if I were a user of this package, I would be very annoyed at their inclusion.

> building the package (cython) 

All dependencies for building the package should be in build or host, not in run.

Reply:
Okay, I needed to make some modifications to my pypi distribution but it looks like I now have it working without these as run-time requirements. I agree this is better, thanks! Does this mean that everything is in order now?